src/hist/axis/__init__.py

```diff
@@ -212,8 +212,8 @@ def __init__(
         *,
         name: str = "",
         label: str = "",
-        growth: bool = False,
         metadata: Any = None,
+        growth: bool = False,
```

Comment:
Not sure why these got reordered, but due to the "*" above, order does not affect users, so it's safe. :)

Reply:
Oh, I reordered them since I saw that order in `Regular`.